### DISCUSSION TITLE
Test with 10000 users

### DISCUSSION THREAD

**Discussion by Elegye** [jlguenego/node-expose-sspi#115] [General]
I see in the README you want to test this library against 10000 users.

For security reasons, I can't tell you for which company I work (at least, French company, with 80 000+ employees).
I can assure you this work with an AD populated by more than 15 000 users (not all employees in the same domain/server).
It is just a bit slow (from 300ms to 1.5s per authentication).

Thanks a lot for your work !

Have a good day.

**Comment by jlguenego** [comment]
cool :smiley:
